pkg/net/net.go

```diff
@@ -74,6 +75,61 @@ func (o *Overlay) loop(ch <-chan EvtState, state event.Emitter) {
 			}
 		}
 	}()
+
+	for {
+		time.Sleep(500 * time.Millisecond) // TODO: decide optimal tick for updating neighbors
+		peer, err := o.n.RandPeer()
+		if err != nil { // TODO: decide what to do errors
```

Comment:
I think we can just log errors.  See `Overlay.log`.

Reply:
`log` and `continue` right?

Reply:
Yep!